src/ArcgisAccount/accountManagerUtils.js

```diff
@@ -0,0 +1,239 @@
+import { UserSession } from '@esri/arcgis-rest-auth';
+import { getSelf } from '@esri/arcgis-rest-portal';
+import { request } from '@esri/arcgis-rest-request';
```

Comment:
`@esri/arcgis-rest-portal` and `@esri/arcgis-rest-request` need to be added as dependencies as well.

Reply:
Updated package.json